examples/computer_vision/byol_pytorch/byol_pytorch.py

```diff
@@ -0,0 +1,294 @@
+"""
+Adapted from https://github.com/lucidrains/byol-pytorch/blob/master/byol_pytorch/byol_pytorch.py
```

Comment:
if this file is the same as the source, i prefer we git clone and checkout a specific commit into the working directory as part of the startup-hook so we don't have to maintain this portion.

Reply:
There were changes, but I was able to add some workarounds for compatibility with the `pip install` version.